apps/src/music/README.md

```diff
@@ -6,7 +6,8 @@
 
 This is the client-side code for **Music Lab**, as instantiated by a `music` level using [Lab2](../lab2/).
 
-**Music Lab** is a powerful lab for creating music using code.  It includes the following features:
+**Music Lab** is a powerful lab for creating music using code. It includes the following features:
```

Comment:
The first few changes are just editor auto-formatting; the new content begins with `## Compilation, Execution, and Playback`

Reply:
I still write with two spaces after a period for almost everything. :)